cmd/nginx-ingress/flags.go

```diff
@@ -16,7 +16,8 @@ import (
 )
 
 const (
-	dynamicSSLReloadParam = "ssl-dynamic-reload"
+	dynamicSSLReloadParam           = "ssl-dynamic-reload"
+	weightChangesWithoutReloadParam = "weight-changes-without-reload"
```

Comment:
I'm only just realising that these are possibly at odds in terms of naming... Maybe it should be `weight-changes-dynamic-reload`? It's probably not a big deal to leave it 🤔

Reply:
yeah we should probably keep them the same, I will change this to `weight-changes-dynamic-reload`